setup.cfg

```diff
@@ -45,7 +45,7 @@ packages = sphinx_rtd_theme
 python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*
 install_requires = 
 	sphinx >=1.6,<6
-	docutils <0.18
+	docutils <0.19
```

Comment:
Is there any reason to not allow docutils 0.19 as well?

AFAIK there are no significant differences in generated HTML between 0.18 and 0.19, so your changes will work fine with 0.19 as well, won't they?

Currently sphinx-rtd-theme is the only reason why I can't upgrade docutils package in Debian/Ubuntu from 0.17, and I would like to upgrade directly to 0.19, rather than from one old version to another old one.

Reply:
https://github.com/readthedocs/sphinx_rtd_theme/issues/1323

Reply:
Okay, got it.